<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>708</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(242, 237, 228);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>491</width>
      <height>711</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(44, 51, 89);
border-radius: 10px;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>130</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font:  75 24pt &quot;Times New Roman&quot;;
color: rgb(242, 237, 228);</string>
     </property>
     <property name="text">
      <string>id</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_id">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>140</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
    background-color: white;
    box-shadow: 2px 0.5px 5px 1px inset black;
    border-radius: 5px;
    width: 70%;
    padding-right: 10px;
    padding-left: 10px;
    text-align: left;
    color: #000;
    font-size:inherit;
    font-family: Arial, Helvetica, sans-serif;  
    font-weight:normal;
    font-variant:normal;
}
QLineEdit::hover{  color: black ;
background-color:orange;}
QLineEdit::focus{  color: black ;
background-color:white;}</string>
     </property>
     <property name="placeholderText">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>180</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font:  75 24pt &quot;Times New Roman&quot;;
color: rgb(242, 237, 228);</string>
     </property>
     <property name="text">
      <string>nom</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_nom">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>190</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
    background-color: white;
    box-shadow: 2px 0.5px 5px 1px inset black;
    border-radius: 5px;
    width: 70%;
    padding-right: 10px;
    padding-left: 10px;
    text-align: left;
    color: #000;
    font-size:inherit;
    font-family: Arial, Helvetica, sans-serif;  
    font-weight:normal;
    font-variant:normal;
}
QLineEdit::hover{  color: black ;
background-color:orange;}
QLineEdit::focus{  color: black ;
background-color:white;}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>NOM</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>230</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font:  75 24pt &quot;Times New Roman&quot;;
color: rgb(242, 237, 228);</string>
     </property>
     <property name="text">
      <string>prenom</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_prenom">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>240</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
    background-color: white;
    box-shadow: 2px 0.5px 5px 1px inset black;
    border-radius: 5px;
    width: 70%;
    padding-right: 10px;
    padding-left: 10px;
    text-align: left;
    color: #000;
    font-size:inherit;
    font-family: Arial, Helvetica, sans-serif;  
    font-weight:normal;
    font-variant:normal;
}
QLineEdit::hover{  color: black ;
background-color:orange;}
QLineEdit::focus{  color: black ;
background-color:white;}</string>
     </property>
     <property name="placeholderText">
      <string>PRENOM</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>290</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font:  75 24pt &quot;Times New Roman&quot;;
color: rgb(242, 237, 228);</string>
     </property>
     <property name="text">
      <string>type_imma</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="type_immatriculation">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>300</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
    background-color: white;
    box-shadow: 2px 0.5px 5px 1px inset black;
    border-radius: 5px;
    width: 70%;
    padding-right: 10px;
    padding-left: 10px;
    text-align: left;
    color: #000;
    font-size:inherit;
    font-family: Arial, Helvetica, sans-serif;  
    font-weight:normal;
    font-variant:normal;
}
QLineEdit::hover{  color: black ;
background-color:orange;}
QLineEdit::focus{  color: black ;
background-color:white;}</string>
     </property>
     <property name="placeholderText">
      <string>type_immatriculation</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>350</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font:  75 24pt &quot;Times New Roman&quot;;
color: rgb(242, 237, 228);</string>
     </property>
     <property name="text">
      <string>Num_imma</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_numImmatriculation">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>360</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
    background-color: white;
    box-shadow: 2px 0.5px 5px 1px inset black;
    border-radius: 5px;
    width: 70%;
    padding-right: 10px;
    padding-left: 10px;
    text-align: left;
    color: #000;
    font-size:inherit;
    font-family: Arial, Helvetica, sans-serif;  
    font-weight:normal;
    font-variant:normal;
}
QLineEdit::hover{  color: black ;
background-color:orange;}
QLineEdit::focus{  color: black ;
background-color:white;}


</string>
     </property>
     <property name="placeholderText">
      <string>numImmatriculation</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_modifier">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>470</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">/*-----QPushButton-----*/
QPushButton
{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 
#92bf91
,stop:0.995192 rgba(0,0,0,0));



	color: #ffffff;
	font-size: 22px;
	font-weight: bold;
	border: 2px solid #9fbdb3;
	border-radius:10px;
	padding: 5px;
	

}

QPushButton::hover
{
background-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(0, 180, 150, 220),stop:1 rgba(50, 220, 200, 220));

	color: #2C3359; 

}
QPushButton::pressed
{background-color: qlineargradient(stop:0 rgba(0, 207, 179, 255),stop:1 rgba(70, 255, 230, 255));

	color: #2C3359; 

}</string>
     </property>
     <property name="text">
      <string>modifier</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>481</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font:  75 24pt &quot;Times New Roman&quot;;
color: rgb(242, 190, 92);
text-decoration: underline;</string>
     </property>
     <property name="text">
      <string>veuillez remplir ce formulaire</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>480</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/plus.svg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../Downloads/icons8-modifier-l'utilisateur-homme-32.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>480</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/plus.svg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../Downloads/icons8-approbation-32.png</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_ajouter">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>470</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">/*-----QPushButton-----*/
QPushButton
{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 
#92bf91
,stop:0.995192 rgba(0,0,0,0));



	color: #ffffff;
	font-size: 22px;
	font-weight: bold;
	border: 2px solid #9fbdb3;
	border-radius:10px;
	padding: 5px;
	

}

QPushButton::hover
{
background-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(0, 180, 150, 220),stop:1 rgba(50, 220, 200, 220));

	color: #2C3359; 

}
QPushButton::pressed
{background-color: qlineargradient(stop:0 rgba(0, 207, 179, 255),stop:1 rgba(70, 255, 230, 255));

	color: #2C3359; 

}</string>
     </property>
     <property name="text">
      <string>Reserver</string>
     </property>
    </widget>
    <zorder>pb_ajouter</zorder>
    <zorder>label_11</zorder>
    <zorder>le_id</zorder>
    <zorder>label_12</zorder>
    <zorder>le_nom</zorder>
    <zorder>label_13</zorder>
    <zorder>le_prenom</zorder>
    <zorder>label_4</zorder>
    <zorder>type_immatriculation</zorder>
    <zorder>label_5</zorder>
    <zorder>le_numImmatriculation</zorder>
    <zorder>pb_modifier</zorder>
    <zorder>label</zorder>
    <zorder>label_15</zorder>
    <zorder>label_16</zorder>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1421</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(44, 51, 89);
border-radius: 10px;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>201</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">/*-----QPushButton-----*/
QPushButton
{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 
#92bf91
,stop:0.995192 rgba(0,0,0,0));



	color: #ffffff;
	font-size: 16px;
	font-weight: bold;
	border: 2px solid #9fbdb3;
	border-radius:10px;
	padding: 5px;
	

}

QPushButton::hover
{
background-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(0, 180, 150, 220),stop:1 rgba(50, 220, 200, 220));

	color: #2C3359; 

}
QPushButton::pressed
{background-color: qlineargradient(stop:0 rgba(0, 207, 179, 255),stop:1 rgba(70, 255, 230, 255));

	color: #2C3359; 

}</string>
     </property>
     <property name="text">
      <string>Gestion de reservation</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>20</y>
       <width>141</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">/*-----QPushButton-----*/
QPushButton
{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 
#92bf91
,stop:0.995192 rgba(0,0,0,0));



	color: #ffffff;
	font-size: 16px;
	font-weight: bold;
	border: 2px solid #9fbdb3;
	border-radius:10px;
	padding: 5px;
	

}

QPushButton::hover
{
background-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(0, 180, 150, 220),stop:1 rgba(50, 220, 200, 220));

	color: #2C3359; 

}
QPushButton::pressed
{background-color: qlineargradient(stop:0 rgba(0, 207, 179, 255),stop:1 rgba(70, 255, 230, 255));

	color: #2C3359; 

}</string>
     </property>
     <property name="text">
      <string>Gestion de piste</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>20</y>
       <width>171</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">/*-----QPushButton-----*/
QPushButton
{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 
#92bf91
,stop:0.995192 rgba(0,0,0,0));



	color: #ffffff;
	font-size: 16px;
	font-weight: bold;
	border: 2px solid #9fbdb3;
	border-radius:10px;
	padding: 5px;
	

}

QPushButton::hover
{
background-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(0, 180, 150, 220),stop:1 rgba(50, 220, 200, 220));

	color: #2C3359; 

}
QPushButton::pressed
{background-color: qlineargradient(stop:0 rgba(0, 207, 179, 255),stop:1 rgba(70, 255, 230, 255));

	color: #2C3359; 

}</string>
     </property>
     <property name="text">
      <string>Gestion deemploye</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>20</y>
       <width>151</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">/*-----QPushButton-----*/
QPushButton
{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 
#92bf91
,stop:0.995192 rgba(0,0,0,0));



	color: #ffffff;
	font-size: 16px;
	font-weight: bold;
	border: 2px solid #9fbdb3;
	border-radius:10px;
	padding: 5px;
	

}

QPushButton::hover
{
background-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(0, 180, 150, 220),stop:1 rgba(50, 220, 200, 220));

	color: #2C3359; 

}
QPushButton::pressed
{background-color: qlineargradient(stop:0 rgba(0, 207, 179, 255),stop:1 rgba(70, 255, 230, 255));

	color: #2C3359; 

}</string>
     </property>
     <property name="text">
      <string>Gestion de visite</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>20</y>
       <width>171</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">/*-----QPushButton-----*/
QPushButton
{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 
#92bf91
,stop:0.995192 rgba(0,0,0,0));



	color: #ffffff;
	font-size: 16px;
	font-weight: bold;
	border: 2px solid #9fbdb3;
	border-radius:10px;
	padding: 5px;
	

}

QPushButton::hover
{
background-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(0, 180, 150, 220),stop:1 rgba(50, 220, 200, 220));

	color: #2C3359; 

}
QPushButton::pressed
{background-color: qlineargradient(stop:0 rgba(0, 207, 179, 255),stop:1 rgba(70, 255, 230, 255));

	color: #2C3359; 

}</string>
     </property>
     <property name="text">
      <string>Gestion de vehicule</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>982</x>
       <y>20</y>
       <width>211</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">/*-----QPushButton-----*/
QPushButton
{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 
#92bf91
,stop:0.995192 rgba(0,0,0,0));



	color: #ffffff;
	font-size: 16px;
	font-weight: bold;
	border: 2px solid #9fbdb3;
	border-radius:10px;
	padding: 5px;
	

}

QPushButton::hover
{
background-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(0, 180, 150, 220),stop:1 rgba(50, 220, 200, 220));

	color: #2C3359; 

}
QPushButton::pressed
{background-color: qlineargradient(stop:0 rgba(0, 207, 179, 255),stop:1 rgba(70, 255, 230, 255));

	color: #2C3359; 

}</string>
     </property>
     <property name="text">
      <string>Gestion de equipement</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableView" name="tab_reservation">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>60</y>
      <width>941</width>
      <height>661</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 
#92bf91
,stop:0.995192 rgba(0,0,0,0));

QMainWindow{
	background-image: url(:/new/prefix1/logo (2).png);

background-repeat: no-repeat;



}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="id_supp">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>540</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{

    background-color: white;

    box-shadow: 2px 0.5px 5px 1px inset black;

    border-radius: 5px;

    width: 70%;

    padding-right: 10px;

    padding-left: 10px;

    text-align: left;

    color: #000;

    font-size:inherit;

    font-family: Arial, Helvetica, sans-serif;  

    font-weight:normal;

    font-variant:normal;

}

QLineEdit::hover{  color: black ;

background-color:orange;}

QLineEdit::focus{  color: black ;

background-color:white;}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_supprimer">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>580</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/*-----QPushButton-----*/
QPushButton
{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 
#f29c94
,stop:0.995192 rgba(0,0,0,0));



	color: #ffffff;
	font-size: 22px;
	font-weight: bold;
	border: 2px solid #cc7f78;
	border-radius:10px;
	padding: 5px;
	

}

QPushButton::hover
{
background-color: #c29c99;

	color: #2C3359; 

}
QPushButton::pressed
{background-color: qlineargradient(stop:0 rgba(0, 207, 179, 255),stop:1 rgba(70, 255, 230, 255));

	color: #2C3359; 

}</string>
    </property>
    <property name="text">
     <string>Supprimer</string>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/new/prefix1/icons8-supprimer-pour-toujours-32 (1).png</normaloff>:/new/prefix1/icons8-supprimer-pour-toujours-32 (1).png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>31</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>540</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 
#92bf91
,stop:0.995192 rgba(0,0,0,0));

font:  75 20pt &quot;Times New Roman&quot;;
color: rgb(44, 51, 89);</string>
    </property>
    <property name="text">
     <string>id</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_rechercher">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>460</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/*-----QPushButton-----*/
QPushButton
{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 
#92bf91
,stop:0.995192 rgba(0,0,0,0));



	color: #ffffff;
	font-size: 22px;
	font-weight: bold;
	border: 2px solid #9fbdb3;
	border-radius:10px;
	padding: 5px;
	

}

QPushButton::hover
{
background-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(0, 180, 150, 220),stop:1 rgba(50, 220, 200, 220));

	color: #2C3359; 

}
QPushButton::pressed
{background-color: qlineargradient(stop:0 rgba(0, 207, 179, 255),stop:1 rgba(70, 255, 230, 255));

	color: #2C3359; 

}</string>
    </property>
    <property name="text">
     <string>rechercher</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_recherche">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>450</y>
      <width>711</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
background-color: rgb(242, 247, 245);
border-radius: 10px;
border: 2px solid #2596be


}
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_filtreRechercheMobilite">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>510</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(218, 237, 226);</string>
    </property>
    <property name="text">
     <string>Filtre recherche</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_Longueur">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>530</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(218, 237, 226);</string>
    </property>
    <property name="text">
     <string>Recherche par longueur</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_Largeur">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>550</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(218, 237, 226);</string>
    </property>
    <property name="text">
     <string>Recherche par largeur</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_rechercher_type">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>570</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(218, 237, 226);</string>
    </property>
    <property name="text">
     <string>Recherche par type</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>460</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(242, 247, 245);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Downloads/icons8-rechercher-un-client-32.png</pixmap>
    </property>
   </widget>
   <zorder>tab_reservation</zorder>
   <zorder>frame</zorder>
   <zorder>frame_2</zorder>
   <zorder>id_supp</zorder>
   <zorder>pb_supprimer</zorder>
   <zorder>label_14</zorder>
   <zorder>pb_rechercher</zorder>
   <zorder>le_recherche</zorder>
   <zorder>label_filtreRechercheMobilite</zorder>
   <zorder>checkBox_Longueur</zorder>
   <zorder>checkBox_Largeur</zorder>
   <zorder>cb_rechercher_type</zorder>
   <zorder>label_2</zorder>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
